diff --git a/node_modules/zwift-mobile-api/src/World.js b/node_modules/zwift-mobile-api/src/World.js
index 6bc3075..2a0a376 100644
--- a/node_modules/zwift-mobile-api/src/World.js
+++ b/node_modules/zwift-mobile-api/src/World.js
@@ -7,7 +7,7 @@ class World {
     }
 
     riders() {
-        return this.request.json(`/relay/developer/worlds/${this.worldId}`)
+        return this.request.json(`/relay/worlds/${this.worldId}`)
     }
 
     riderStatus(playerId) {
@@ -15,7 +15,7 @@ class World {
             throw new Error('A player id is required - world.riderStatus(playerId)')
         }
 
-        return this.request.json(`/relay/developer/worlds/${this.worldId}/players/${playerId}`);
+        return this.request.json(`/relay/worlds/${this.worldId}/players/${playerId}`);
     }
 
     segmentResults() {
diff --git a/node_modules/zwift-mobile-api/src/getAccessToken.js b/node_modules/zwift-mobile-api/src/getAccessToken.js
index 4b5b18e..1ec9407 100644
--- a/node_modules/zwift-mobile-api/src/getAccessToken.js
+++ b/node_modules/zwift-mobile-api/src/getAccessToken.js
@@ -1,7 +1,7 @@
 ﻿const axios = require('axios')
 const qs = require('qs')
 
-const SECURE_URL = process.env.ZwiftSecureUrl || 'https://secure.zwift.com/auth/realms/zwift/tokens/access/codes';
+const SECURE_URL = process.env.ZwiftSecureUrl || 'https://secure.zwift.com/auth/realms/zwift/protocol/openid-connect/token';
 
 module.exports = function getAccessToken(username, password, refreshToken = null) {
     let data
